packages/terra-notification-dialog/src/terra-dev-site/doc/example/NotificationDialogWithLongText.jsx

```diff
@@ -27,85 +27,85 @@ class NotificationDialogWithLongText extends React.Component {
-      'Integer congue feugiat ultricies.',
-      ' Nunc non mauris sed tellus cursus vestibulum nec quis ipsum.',
-      'Vivamus ornare magna justo, et volutpat tortor congue ut. Nulla facilisi.',
-      ' Cras in venenatis turpis. Nullam id odio justo. Etiam vehicula lectus vel purus consectetur cursus id sit amet diam.',
-      'Donec facilisis dui non orci hendrerit pharetra. Suspendisse blandit dictum turpis, in consectetur ipsum hendrerit eget.',
-      'Nam vehicula, arcu vitae egestas porttitor,',
-      'turpis nisi pulvinar neque, ut lacinia urna purus sit amet elit.',
-      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. ',
-      'Maecenas molestie in lorem vel facilisis. Quisque ac enim nec lectus malesuada faucibus.',
-      'Integer congue feugiat ultricies.',
-      ' Nunc non mauris sed tellus cursus vestibulum nec quis ipsum.',
-      'Vivamus ornare magna justo, et volutpat tortor congue ut. Nulla facilisi.',
-      ' Cras in venenatis turpis. Nullam id odio justo. Etiam vehicula lectus vel purus consectetur cursus id sit amet diam.',
-      'Donec facilisis dui non orci hendrerit pharetra. Suspendisse blandit dictum turpis, in consectetur ipsum hendrerit eget.',
-      'Nam vehicula, arcu vitae egestas porttitor',
-      'turpis nisi pulvinar neque, ut lacinia urna purus sit amet elit.',
-      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. ',
-      'Maecenas molestie in lorem vel facilisis. Quisque ac enim nec lectus malesuada faucibus.',
-      'Integer congue feugiat ultricies.',
-      ' Nunc non mauris sed tellus cursus vestibulum nec quis ipsum.',
-      'Vivamus ornare magna justo, et volutpat tortor congue ut. Nulla facilisi.',
-      ' Cras in venenatis turpis. Nullam id odio justo. Etiam vehicula lectus vel purus consectetur cursus id sit amet diam.',
-      'Donec facilisis dui non orci hendrerit pharetra. Suspendisse blandit dictum turpis, in consectetur ipsum hendrerit eget.',
-      'Nam vehicula, arcu vitae egestas porttitor,',
-      'turpis nisi pulvinar neque, ut lacinia urna purus sit amet elit.',
-      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. ',
-      'Maecenas molestie in lorem vel facilisis. Quisque ac enim nec lectus malesuada faucibus.',
-      'Integer congue feugiat ultricies.',
-      ' Nunc non mauris sed tellus cursus vestibulum nec quis ipsum.',
-      'Vivamus ornare magna justo, et volutpat tortor congue ut. Nulla facilisi.',
-      ' Cras in venenatis turpis. Nullam id odio justo. Etiam vehicula lectus vel purus consectetur cursus id sit amet diam.',
-      'Donec facilisis dui non orci hendrerit pharetra. Suspendisse blandit dictum turpis, in consectetur ipsum hendrerit eget.',
-      'Nam vehicula, arcu vitae egestas porttitor,',
-      'turpis nisi pulvinar neque, ut lacinia urna purus sit amet elit.'];
+    const message = [`Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```

Comment:
Does this need to be an array at all after updating to use back ticks?

Reply:
Whoops, nope. Thanks.